src/server/services/pickup/configure-server.js

```diff
@@ -1,25 +1,107 @@
 import Rcon from 'modern-rcon';
 import debug from 'debug';
+import config from 'config';
+import fs from 'fs';
+import sleep from 'await-sleep';
 
 const log = debug('TF2Pickup:server');
 
 /**
+ * Change server map.
+ *
+ * @param {Object} connection - RCON connection object to server.
+ * @param {Object} map - Map name.
+ */
+async function changeMap(connection, map) {
+  const maplist = await connection.send('maps *');
+
+  if (maplist.indexOf(map) === -1) {
+    throw new Error(`Server does not have map ${map}`);
+  } else {
+    log(`Changing server map to ${map}`);
+    await sleep(3 * 1000);
+    await connection.send(`changelevel ${map}`);
+  }
+}
+
+/**
+ * Change server config.
+ *
+ * @param {Object} connection - RCON connection object to server.
+ * @param {String} cfg - CFG file to execute.
+ */
+function executeConfig(connection, cfg) {
+  const nodePath = process.env.NODE_PATH;
+  const configPath = `${nodePath}/../node_modules/@tf2-pickup/tf2-configs/dist/${cfg}.cfg`;
+  const configFile = fs.readFileSync(configPath, 'utf8');
+  const configLines = configFile.split('\n');
+
+  configLines.forEach(async (line) => {
+    log(line);
```

Comment:
We don't need to log every line of the config file, we can just log that we are executing a config, that should be enough

Reply:
That's just for the test to see check if configs were reading and executing correctly. I will remove it later.